app/templates/_base_page.html

```diff
@@ -49,7 +49,20 @@
 
 
 {% block content %}
-  {% include "toolkit/flash_messages.html" %}
+  {% block flashMessages %}
+    {% with
+       messages = get_flashed_messages(with_categories=True),
+       titles = {"error": "There is a problem", "success": "Success"}
```

Comment:
Is "Success" now going to be displayed above the usual success `message` as a title?

Reply:
Yes.

It should look something like this:

![Success. You have successfully changed your password.](https://user-images.githubusercontent.com/503614/84255081-04fe0380-ab0a-11ea-9397-692776d8fe0a.png)

Reply:
Interesting! Is this a Design System pattern/recommendation, or our own decision?

Reply:
Now that I think about it, I can't remember 😅

I think it might have been something I did ages ago when doing a spike into alerts without thinking too hard about it.

Maybe we should take it out then?